pyuploadcare/secure_url.py

```diff
@@ -1,18 +1,20 @@
 import binascii
 import hashlib
 import hmac
+import re
 import time
 import warnings
 from abc import ABC, abstractmethod
 from typing import Optional
+from urllib.parse import quote_plus, urlparse
 
 
 class BaseSecureUrlBuilder(ABC):
     @abstractmethod
-    def build(self, uuid: str, wildcard: bool = False) -> str:
+    def build(self, uuid_or_url: str, wildcard: bool = False) -> str:
```

Comment:
here and everywhere rename `uuid_or_url` to `handle`?

Reply:
Unsure. `uuid_or_url` is ugly; `handle` is ambiguous.